suites/fs/basic/tasks/asok_dump_tree.yaml

```diff
@@ -0,0 +1,4 @@
+tasks:
+- cephfs_test_runner:
```

Comment:
In fs/basic the mounts are defined in the tasks (like in the cfuse_*) ones.  So this is failing to run because it doesn't have any mounts.  (Different from the ones in fs/recovery which have the mounts defined elsewhere fs/recovery/mounts)